rust/types/src/trade.rs

```diff
@@ -11,3 +11,47 @@ pub struct Trade {
     pub timestamp: i64,
     pub is_buyer_maker: bool,
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TradeUpdate {
+    /// Event type
```

Comment:
```suggestion
    /// Event type.
```
Given it is doc entry. Ditto below.

Reply:
I don't think we have this documented by the way. Also we just add the type here. And perhaps it should be conditional on the `ws` feature.